apache/trafficcontrol
# TP: fixes any map ds form and adds anonymous blocking to steering form

## What does this PR (Pull Request) do?

- [x] This PR fixes #3405 and fixes #1943 and fixes #4102 

This PR actually fixes 3 issues. I could have done 3 PR's but figured since each is so small, I'd just submit one PR.

- None of these forms were documented, therefore, making changes does not require a documentation change.
- The existing UI tests server as an adequate regression test of each delivery service form.

## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?
Issue #3405:
- ensure that the anonymous blocking field has been added to the STEERING* delivery service forms.

Issue #1943:
- ensure that the Routing Configuration Settings section has been removed from the ANY_MAP delivery service form.

Issue #4102 
- ensure that the delivery service URLs section has been removed from the ANY_MAP delivery service form. 

## If this is a bug fix, what versions of Traffic Control are affected?
- master (e3c48b6)

## The following criteria are ALL met by this PR
